File: DayThree/while_loop_prime_number.py
my_num = int(input("Enter a number: "))

num_of_factor = 0
divisor = 1
while divisor <= my_num:
  if my_num % divisor == 0:
    num_of_factor += 1
  divisor += 1

if num_of_factor == 2:
  print(my_num, "is a prime number")
else:
  print(my_num, "is not a prime number")

File: DayThree/while_loop_prime_number.swift
print("Enter the number:", terminator: " ")
let userInput = readLine()
let numberInput = userInput!
let myNum = Int(numberInput) ?? 0

var numberOfFactor: Int = 0

var divisor: Int = 1
while (divisor <= myNum) {
  if myNum % divisor == 0 {
    numberOfFactor += 1
  }
  divisor += 1
}

if numberOfFactor == 2 {
  print("\(myNum) is a prime number!")
} else {
  print("\(myNum) is not a prime number :D")
}
